core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -501,6 +501,18 @@
 			'eval'                    => array('maxlength'=>255, 'tl_class'=>'w50'),
 			'sql'                     => "varchar(255) NOT NULL default ''"
 		),
+		'disposition' => [
+			'label'                   => &$GLOBALS['TL_LANG']['tl_content']['disposition'],
+			'exclude'                 => true,
+			'inputType'               => 'select',
+			'options' => [
+				\Symfony\Component\HttpFoundation\ResponseHeaderBag::DISPOSITION_ATTACHMENT,
+				\Symfony\Component\HttpFoundation\ResponseHeaderBag::DISPOSITION_INLINE
+			],
+			'reference'               => &$GLOBALS['TL_LANG']['MSC']['disposition'],
+			'eval'                    => ['tl_class' => 'w50'],
+			'sql'                     => "varchar(32) NOT NULL default '" . \Symfony\Component\HttpFoundation\ResponseHeaderBag::DISPOSITION_ATTACHMENT . "'"
```

Comment:
@leofeyer what is best practise here within Contao? use
```php
'sql' => "varchar(32) NOT NULL default ''"
```
together with
```php
'default' => ResponseHeaderBag::DISPOSITION_ATTACHMENT
```
? Usually the defaults within Contao are an empty string or null. Even if the field stays empty, the default will still be `attachment` due to the backwards compatible implementation within `sendFileToBrowser`.

Also should one use the FQCN within the DCA or use
```php
use Symfony\Component\HttpFoundation\ResponseHeaderBag;
```
?

Reply:
FQCN is good here. It's a config array, not a class.

Reply:
Sure, but is the usage of "use" limited to classes? I always use "use" in DCAs as well to keep them concise.

Reply:
I find it way harder to debug. That's why I prefer FQCN in DCA's. Matter of preference, there's no rule and I don't care really.

Reply:
So? I don't see why that would be a problem? Every field has a gallery list of 4 items by default, regardless of whether it's a gallery element or not.

Reply:
That's a bad example, because storing `4` instead of `0` does not affect the database size. But here are some fields we do need to fix:

 * `tl_content.sliderSpeed` stores `300` in every row (should be `0`)
 * `tl_module.queryType` stores `and` in every row (should be `''`)
 * `tl_module.searchType` stores `simple` in every row (should be `''`)
 * `tl_page.sitemap` stores `map_default` in every row (should be `''`) 

However, we should open a separate ticket hereto.

Reply:
See contao/contao#56

Reply:
As discussed in Mumble on September 27th, we are currently not moving the defaults to the SQL definition, therefore the default needs to go into the `'default'` value.